pyvista/core/cells.py

```diff
@@ -0,0 +1,79 @@
+"""pyvista wrapping of vtkCellArray."""
+import numpy as np
+from vtk.util.numpy_support import numpy_to_vtkIdTypeArray, vtk_to_numpy
+from vtk import vtkCellArray
+
+import pyvista
+
+
+def numpy_to_idarr(ind, deep=False, return_ind=False):
+    """Safely convert a numpy array to a vtkIdTypeArray"""
+    try:
+        ind = np.array(ind)
```

Comment:
To avoid unnecessary copies I'd suggest
```suggestion
        ind = np.asarray(ind)
```